tools/mdfileconverter/md_converter.xml

```diff
@@ -45,11 +47,17 @@
             <param name="input_file" value="traj.dcd" ftype="dcd"/>
             <output name="output" file="traj.trr"/>
         </test>
+        <test>
+            <param name="conversion" value="traj" />
+            <param name="output_format" value="netcdf" />
+            <param name="input_file" value="traj.trr" ftype="trr"/>
+            <output name="output" file="traj.netcdf" compare="sim_size"/>
```

Comment:
can you use an assertion with `has_size` here? https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-tests-test-output-assert-contents

this saves needing a large test file.

Reply:
Trying it out. Then should I remove the example output file?

Reply:
@simonbray Does this work with binary outputs. Tried quite a liberal delta and still get this error
```
JobOutputsError: History item  different than expected, difference (using diff):
( /tmp/tmp_jRy4Gtraj.netcdf v. /tmp/tmpxtdTLXtraj.netcdf )
Binary data detected, not displaying diff
```

Reply:
@chrisbarnettster make sure you are using latest Galaxy ... or even the new-beta 20.01

Reply:
Thanks I used the latest branch and looks good!  `planemo test --galaxy_branch release_20.01`